tests/css_parens/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -128,6 +128,78 @@ a {
       );
   padding-top: var(  --paddingC  );
 }
+
+.bar {
+    prop1: attr(data-foo);
+    prop2: attr( data-foo );
+    prop3: attr(  data-foo  );
+    prop4: attr(
+        data-foo
+    );
+    prop5: attr(
+        data-foo
+    );
+    prop6:
+        attr(data-foo);
+    prop7:
+        attr(
+            data-foo
+        );
+    prop8:
+    attr(
+    data-foo
+    )
+    ;
+    prop9:"("attr(title)")";
```

Comment:
This parsed as `value-comma_group`, seems it is invalid and should be `value-list_group` and we should implement logic

Reply:
We should probably open a tracking issue

Reply:
@suchipi done https://github.com/prettier/prettier/issues/3842